src/main/java/seedu/guest/model/guest/Request.java

```diff
@@ -0,0 +1,68 @@
+package seedu.guest.model.guest;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.commons.util.AppUtil.checkArgument;
+
+/**
+ * The class Requests.
+ */
+public class Request {
+    public static final String MESSAGE_CONSTRAINTS = "The request should not be null";
+    public final String value;
+
+    /**
+     * The constructor with no specific request.
+     */
+    public Request() {
+        value = "Nothing";
+    }
+
+    /**
+     * The constructor with specific request.
+     * @param str the specific request
+     */
+    public Request(String str) {
```

Comment:
Should we follow the same documentation-style for the other field classes as well for the constructor?

Reply:
changed